packages/react-slate-core/src/public/View.js

```diff
@@ -10,12 +10,13 @@ import { getStyleProps } from '../utils/style';
 type Props = {
   style?: Style,
   children: mixed,
+  render?: *,
```

Comment:
Don't feel very good about this prop. It seems like this is exposing some internal details (effectively the internal node instance). And it's confusing what's the difference between `render` and `children` is.

Maybe you should accept `children` as function instead or another component altogether.

Reply:
It's an escape hatch. Also this props is not documented so typical user should not use it. Accepting `children` as a function __it not going to work__, because what I need to do is different what `children` as a function are capable of doing.